File: Level01/03.평균 구하기/getMean.js
function average(array) {
    var sum = 0;  
    for (var idx in array) {
        sum += array[idx];
    }
    return sum / array.length;
}

// 아래는 테스트로 출력해 보기 위한 코드입니다.
var testArray = [5,3,4] 
console.log("평균값 : " + average(testArray));

File: Level01/03.평균 구하기/GetMean.java
public class GetMean {

    public int getMean(int[] array) {
        long sum = 0L;
        for (int value : array) {
            sum += value;          
        }
        return (int) (sum / array.length);
    }

    public static void main(String[] args) {
        int x[] = {5, 4, 3};
        GetMean getMean = new GetMean();
        // 아래는 테스트로 출력해 보기 위한 코드입니다.
        System.out.println("평균값 : " + getMean.getMean(x));
    }
}
